ci-operator/jobs/integr8ly/integreatly-operator/integr8ly-integreatly-operator-MGDAPI-3785-sts-postsubmits.yaml

```diff
@@ -0,0 +1,108 @@
+postsubmits:
+  integr8ly/integreatly-operator:
+  - agent: kubernetes
+    always_run: true
+    branches:
+    - ^MGDAPI-3785-sts$
+    cluster: build01
+    decorate: true
+    decoration_config:
+      skip_cloning: true
+    labels:
+      ci-operator.openshift.io/is-promotion: "true"
+      ci.openshift.io/generator: prowgen
+    max_concurrency: 1
+    name: branch-ci-integr8ly-integreatly-operator-MGDAPI-3785-sts-images
```

Comment:
the branch name needs to be in lower case

Reply:
/hold

Reply:
https://github.com/RHCloudServices/integreatly-help/pull/809/files#diff-724bd6fc5716e9048929e74dbe1529ad56ee744e4f5d799fd568ea9345b23fb4R4